shell/components/ResourceDetail/index.vue

```diff
@@ -229,7 +242,9 @@ export default {
     }
 
     // Ensure common properties exists
-    model = await store.dispatch(`${ inStore }/cleanForDetail`, model);
+    try {
+      model = await store.dispatch(`${ inStore }/cleanForDetail`, model);
+    } catch (e) {}
```

Comment:
Are we intentionally performing a noop in the `catch` block?

Reply:
Added a push to errors